test/test_states/test_choices.js

```diff
@@ -533,15 +535,17 @@ describe("states.choice", function() {
                             .inputs(null, '4', '3', '3', '3')
                             .check.reply([
                                 "Hello.",
-                                "1. Plugh",
-                                "2. Back"
+                                "1. Plu",
+                                "2. Pli",
+                                "3. Plo",
+                                "4. Back"
```

Comment:
Added a few choices here that comes out to 21 chars (see test setup) long to ensure they fit on one page, with previous code they wouldn't have as it would have been calculating with double-digit numbers.